analyzers/packaging/SonarAnalyzer.CSharp.Styling.nuspec

```diff
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>SonarAnalyzer.CSharp.Styling</id>
+    <version>9.24.0.0</version>
```

Comment:
The script that sets version doesn't work. You need to update the expected match count

Reply:
I updated the behavior of `set-version.ps1` script.
I went for what seemed to me the easiest to do.

Reply:
Why not to add release notes in the nuspec, to keep it all aligned?

Reply:
Since it is internal and we will never mention the styling change in the release note, I thought it was not necessary to keep it.

I don't mind adding back.

Reply:
We might mention it in the future. And it gives us one more backlink too. It's fine to have release notes as they are there now